.github/workflows/fidelity-tests.yml

```diff
@@ -36,9 +36,17 @@ jobs:
       run: npm run bootstrap
 
     - name: Build packages
-      run: npm run build
+      run: |
+       ./node_modules/.bin/lerna run --scope @google/3dom build --stream
+       ./node_modules/.bin/lerna run --scope @google/model-viewer build --stream
+       ./node_modules/.bin/lerna run --scope @google/model-viewer-render-fidelity-tools build --stream
+    
+    # TODO: without build legacy support, some scenarios give wrong metrics in fidelity test workflow. 
+    # There's some tricky bug here but we don't know how to solve yet. 
+    - name: Generate legacy bundles
+      run: npm run build:legacy-support
```

Comment:
Build legacy-support stops before fidelity test stops it from giving crazy metrics. Model-viewer's renderer's page use the latest version of model-viewer to render golden and should have nothing to do with legacy support (https://github.com/google/model-viewer/blob/master/packages/render-fidelity-tools/test/renderers/model-viewer/index.html#L51), so this is really confusing. Any idea on how this could happen would be super helpful.